ajax/libs/simpleCookie/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "simplecookie",
+  "description": "A small javascript utility to make handling cookies simple",
+  "filename": "simpleCookie.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tevko/simpleCookie.git"
+  },
+  "keywords": [
+    "javascript",
+    "cookie"
+  ],
+  "author": "Tim Evko",
+  "license": "MIT",
```

Comment:
License is listed as ISC in the NPM package: https://www.npmjs.com/package/simplecookie